.changes/1.3.0.md

```diff
@@ -1,2 +1 @@
 ## dbt-trino 1.3.0 - October 13, 2022
-
```

Comment:
unrelated

Reply:
Actually this is related to the version bump. this is done by pre-commit check.